reactor-core/src/test/java/reactor/core/publisher/FluxFirstWithValueTest.java

```diff
@@ -67,7 +67,7 @@ void onlyErrorOrCompleteEmptyEmitsError() {
 				.expectErrorSatisfies(e -> {
 					assertThat(e).isInstanceOf(NoSuchElementException.class);
 					assertThat(e.getMessage()).isEqualTo("All sources completed with error or without values");
```

Comment:
Not related to this issue **per se** but can use the opportunity to change `assertThat(e.getMessage())` to `assertThat().hasMessage()` and variants.
Otherwise LGTM